### ISSUE TITLE
Incorrect name displayed by rocket.cat bot when archiving or unarchiving discussions using slash commands

### ISSUE THREAD

**Issue by mayank6725** [RocketChat/Rocket.Chat#31730]
## Issue(s)
While using slash commands `/archive` and `/unarchive` to archive and unarchive discussions, after successfully archiving or unarchiving, rocket.cat bot displays incorrect name which can be misleading. Since we can have multiple discussions with same name within a channel, the ids displayed instead of name was to uniquely identify each discussion. However, in my opinion notifying users with the id of that discussion is not a feasible solution.

## Supporting proof

https://github.com/RocketChat/Rocket.Chat/assets/69050987/ee4227cd-29ce-4167-a4f6-d0d30679256c

## Steps to reproduce
1. Goto any room other than the Discussion room which you want to archive
2. Try typing `/archive #discussion_name` in the chat
3. rocket.cat bot displays successfully archived but with a incorrect name, specifically the _id of that discussion room
4. Similarly try the steps for `/unarchive #discussion_name`

**Comment by Himanshu-kiran** [comment]
But in my case its working properly that is archiving unarchiving with correct name

**Comment by mayanksaxena007** [comment]
Hi @Himanshu-kiran,
Are you sure you did try with archiving discussions? It happens only in case of discussions. Since multiple discussions can have same name, it shows id instead to uniquely identify discussions.

**Comment by mayank6725** [comment]
Hi @Himanshu-kiran,
Are you sure you did try with archiving discussions? It happens only in case of discussions. Since multiple discussions can have same name, it displays id instead to uniquely identify them.

**Comment by Himanshu-kiran** [comment]
> Hi @Himanshu-kiran, Are you sure you did try with archiving discussions? It happens only in case of discussions. Since multiple discussions can have same name, it displays id instead to uniquely identify them.

Yes, you are right. It works in other channel but not in discussion channel. You have already raised pr with issue no #31729, right ?
